src/main/java/core/basesyntax/database/DataBase.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.database;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class DataBase {
+    public enum Operation {
+        BALANCE("b"),
+        SUPPLY("s"),
+        PURCHASE("p"),
+        RETURN("r");
+
+        private String code;
+
+        Operation(String code) {
+            this.code = code;
+        }
+
+        public String getCode() {
+            return code;
+        }
+    }
+
+    public static final List<String> listDb = new ArrayList<>();

```

Comment:
Remove it from here. There is no connection between DB and enum Operation
```suggestion
```

Reply:
not fixed